bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/channel_types.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solax"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="frequency">
+		<item-type>Number:Frequency</item-type>
+		<label>Electric Frequency</label>
+		<description>Electric frequency of the input/output voltage of the inverter</description>
```

Comment:
I don't remember the latest version, but the now removed channel override had:
```suggestion
		<description>Frequency of the electricity to/from the inverter</description>
```

That version seems more correct.